### PR TITLE
Improve zOS runs when remote LDAP is down

### PR DESCRIPTION
**Pull request by kristip17**
If we're running zOS, the local ApacheDS will not work. Try to setup as many remote LDAPs are available instead of bailing at the first failure. This builds on my previous fix to force zOS runs to stay remote, but resulted in NPEs setting the remote server names because we stop pulling consul servers when we hit an error.

RTC 287143

### CODE REVIEW

**Comment by kristip17** [comment]
#build

File: dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java [hunk 1]
```diff
@@ -137,25 +137,29 @@
...
     static {
 
         /*
          * Determine whether we are running remote or locally.
          */
         boolean uselocalLDAP = isLocalLdapExpectedToBeUsed();
 
+        String os = System.getProperty("os.name").toLowerCase();
+        isZOS = os.startsWith("z/os");
+
```
Comment (Azquelt) [inline dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java]: In other FAT tests, I can see we're doing `LocalMachine.getInstance().getOperatingSystem()` to do this sort of check which might be more reliable.

Comment (kristip17) [reply dev/fattest.simplicity/src/componenttest/topology/utils/LDAPUtils.java]: I'll switch it.

**Comment by kristip17** [comment]
Build failure was 1 ldap_fat read time out failure which is a known intermittent issue. I also ran a z/OS build with ldap_fat and it passed successfully -- https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_O9YnoCguEeyWeMDU9jagxQ

**Review by jvanhill — APPROVED** [review]

**Comment by kristip17** [comment]
#libby

**Pull request merged**